plugins/@grouparoo/postgres/.eslintrc.js

```diff
@@ -0,0 +1,17 @@
+module.exports = {
+  env: {
+    es2021: true,
+    node: true,
+  },
+  extends: ["eslint:recommended", "plugin:@typescript-eslint/recommended"],
+  parser: "@typescript-eslint/parser",
+  parserOptions: {
+    ecmaVersion: 12,
+    sourceType: "module",
+  },
+  plugins: ["@typescript-eslint"],
+  rules: {
+    "@typescript-eslint/consistent-indexed-object-style": ["warn", "record"],
+  },
+  ignorePatterns: ["/dist/", "*.js"],
+};
```

Comment:
Initial eslint config idea.

Reply:
Perhaps this config moves into `@grouparoo/app-templates` and we can synlink it into all future plugins

Reply:
Yeah I'd like to see how we can share these core rules